ring/func/asyncio.py

```diff
@@ -463,6 +463,7 @@ def dict(
             storage_class = fsync.PersistentDictStorage
         else:
             storage_class = fsync.ExpirableDictStorage
+        storage_class.maxsize = maxsize
```

Comment:
I assumed
`if storage_class is None:`
means that we're in a 'default' mode so we're in the somewhat private zone... what about `_maxsize`?

Reply:
The point is not about the naming rule. The next code will make problem.

```python
# maxsize of f is 512 here because fsync.PersistentDictStorage.maxsize is 512
@ring.dict(maxsize=512)
def f():
   ...

# maxsize of both f and g are 128 now becasue fsync.PersistentDictStorage is 128
@ring.dict(maxsize=128)
def g():
   ...
```

We shouldn't set any kind of class variable for decorator parameters.

Reply:
I totally missed that. What a shame. I'll figure out a better way.